UKHomeOffice/cerberus-service
# COP-12326 Added front line action to displayed task outcome

## Description
Displayed the front line outcome on completed tasks if present
![image](https://user-images.githubusercontent.com/12000340/200550568-eeb717e2-622c-474a-acb5-eb1fb9449f90.png)

## To Test
1. Complete a task
2. View the task in the completed tab and confirm it has the correc tag (dismissed, completed, or a target outcome)

## Developer Checklist

\* Required
